### DISCUSSION TITLE
Pocketbase Only Expanding Some Relations

### DISCUSSION THREAD

**Discussion by BobSnrub** [pocketbase/pocketbase#2039] [Q&A]
So I have a Products collection that has 4 relations: prod_details, prod_images, prod_instructions, and prod_materials. I am able to get the first two to expand but the two others do not.  I have modified the relations to see if it had something to do with the relation options but nothing seemed to make any difference. Maybe you can only expand up to two relations per collection? Please lemme know if you know what is happening and/or any tips on my data structure, thanks.  Here are some pics of me trying to expand the collections:

prod_details:
![image](https://user-images.githubusercontent.com/56611572/224454951-99ef0452-c99c-4121-b490-e7e26fe5fb77.png)

prod_materials:
![image](https://user-images.githubusercontent.com/56611572/224455120-cbc28676-64c7-40bf-8e60-0874fac3d6b7.png)

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand your collections structure.

Make sure that the request client satisfies the View API rule of the related collection(s).

If that doesn't help, please export your collections as JSON (Admin UI > Settings > Export collections).

**Reply by BobSnrub** [reply]
Ok, it was the second thing you mentioned there.  I hadn't made any new collections in a while and forgot that collections are set to admin only by default.  Thanks Gani!
